app/models/host.rb

```diff
@@ -1881,5 +1881,16 @@ def read_only_storages
     storages.where(:host_storages => {:read_only => true})
   end
 
+  def archived?
+    ems_id.nil?
+  end
+  alias_method :archived, :archived?
```

Comment:
`VmOrTemplate` has both `archived?` and `archived` https://github.com/ManageIQ/manageiq/blob/master/app/models/vm_or_template.rb#L1376 and I wanted to keep similar behavior with both `Host` and `VmOrTemplate`